src/main/kotlin/com/faforever/userservice/domain/UserService.kt

```diff
@@ -119,7 +123,15 @@ class UserService(
                     .flatMap<LoginResult> { ban ->
                         log.debug("User '$usernameOrEmail' is banned by $ban")
                         hydraService.rejectLoginRequest(challenge, GenericError(HYDRA_ERROR_USER_BANNED))
-                            .map { LoginResult.UserBanned(it.redirectTo, ban) }
+                            .map {
+                                LoginResult.UserBanned(
+                                    UriComponentsBuilder.fromUriString("/banned")
+                                        .queryParam("expiration", if (ban.expiresAt != null) DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(ban.expiresAt) else null)
```

Comment:
static import `ISO_OFFSET_DATE_TIME` so line gets a bit shorter

Reply:
Got it